src/code/stores/simulation-store.coffee

```diff
@@ -81,13 +82,22 @@ SimulationStore   = Reflux.createStore
     @settings.graphHasCollector = @_updateGraphHasCollector()
     @notifyChange()
 
+  _updateUnitNames: ->
+    if @settings.duration is 1
+      @settings.timeUnitOptions = ({name: TimeUnits.toString(unit, false), unit: unit} for unit in TimeUnits.units)
+      @settings.stepUnitsName = TimeUnits.toString(@settings.stepUnits, false)
+    else
+      @settings.timeUnitOptions = ({name: TimeUnits.toString(unit, true), unit: unit} for unit in TimeUnits.units)
+      @settings.stepUnitsName = TimeUnits.toString(@settings.stepUnits, true)
+
```

Comment:
This seems like it could be simplified to something like
```
pluralize = @settings.duration isnt 1
@settings.timeUnitOptions = ({name: TimeUnits.toString(unit, pluralize), unit: unit} for unit in TimeUnits.units)
@settings.stepUnitsName = TimeUnits.toString(@settings.stepUnits, pluralize)
```
just because I worry that the repetition might lead to errors changing something in one place and forgetting it in the other.